core/src/main/java/tc/oc/pgm/tablist/PlayerOrder.java

```diff
@@ -64,24 +64,25 @@ public int compare(MatchPlayer ma, MatchPlayer mb) {
     if (aStaff && !bStaff) return -1;
     else if (bStaff && !aStaff) return 1;
 
-    // Compare the nicknames of 'a' and 'b' if both are nicked, skip group permission check
+    // Compare the nicknames of 'a' and 'b'. If both are nicked, skip group permission check
     if (aNick != null && bNick != null) return aNick.compareToIgnoreCase(bNick);
 
     // If players have different permissions, the player with the highest ranked perm
-    // that the other one does't have is first. Disguised players effectively have no perms.
+    // that the other one doesn't have is first. Disguised players effectively have no perms.
     for (Config.Group group : PGM.get().getConfiguration().getGroups()) {
       Permission permission = group.getPermission();
       boolean aPerm = a.hasPermission(permission);
       boolean bPerm = b.hasPermission(permission);
```

Comment:
```suggestion
      boolean aPerm = a.hasPermission(permission) && aNick == null;
      boolean bPerm = b.hasPermission(permission) && bNick == null;
```
If just included here, makes the follow-up if simpler